ReportingServicesTools/Functions/Admin/Set-RsDatabase.ps1

```diff
@@ -38,6 +38,20 @@ function Set-RsDatabase
         .PARAMETER DatabaseServerName
             Specify the database server name. (e.g. localhost, MyMachine\Sql2016, etc.)
 
+        .PARAMETER Encrypt
+            Specify the encryption type to use when connecting to SQL Server.
+            Accepted values: Mandatory, Optional, Strict.
+            If supported, but not specified, the default value is Mandatory.
```

Comment:
please enhance the comment mentioning what need to support it

If supported by?

Reply:
By "If supported" I meant having the required version of SQLServer PowerShell module installed to use this parameter (as mentioned on the next line) .
Tried to make the comment more comprehensible and added `IMPORTANT:` warning, since the default value of this parameter is the most significant change in the `Invoke-Sqlcmd` (as well as the related scripts) behavior.